### PR TITLE
Use travis container-based infrastructure

### PR DESCRIPTION
**Pull request by pgiraud**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,23 +3,20 @@ language: python
...
-- sudo -u postgres psql -d lux -c 'GRANT SELECT ON spatial_ref_sys TO "www-data";'
-- sudo -u postgres psql -d lux -c 'GRANT ALL ON SCHEMA geov3 TO "www-data";'
-- sudo -u postgres psql -d lux -c 'GRANT ALL ON SCHEMA geov3_static TO "www-data";'
-- sudo -u postgres psql -d lux -c 'GRANT ALL ON geometry_columns TO "www-data";'
+sudo: false
 
-- sudo mkdir -p /srv/tomcat/tomcat1/webapps /var/www/vhosts/luxembourg-geomapfish/conf
-- sudo chmod g+w,o+w /srv/tomcat/tomcat1/webapps /var/www/vhosts/luxembourg-geomapfish/conf
+install:
+  - createdb -E UTF8 -T template0 template_postgis -U postgres
```
Comment (sbrunner) [inline .travis.yml]: Indentation ?

**Comment by pgiraud** [comment]
The travis build fails, but I still don't understand why.

**Comment by sbrunner** [comment]
Previously you add `apache` to `DISABLE_BUILD_RULES` re adding it should fix Travis...

**Comment by pgiraud** [comment]
Indeed. Thanks.
I struggled a bit with this initial syntax error making travis builds failing to start silently.

**Comment by pgiraud** [comment]
Please review.

**Comment by sbrunner** [comment]
Looks good :-)

**Pull request merged**